src/Components/WebAssembly/DevServer/src/Microsoft.AspNetCore.Components.WebAssembly.DevServer.csproj

```diff
@@ -14,14 +14,15 @@
     <!-- Disable the default runtimeconfig.json in favor of the one that we create which contains the correct
     framework references. Workaround for issues launching Blazor WASM apps with IIS Express. -->
     <GenerateRuntimeConfigurationFiles>false</GenerateRuntimeConfigurationFiles>
+    <AddPublicApiAnalyzers>false</AddPublicApiAnalyzers>
+    <Nullable>enable</Nullable>
   </PropertyGroup>
 
   <ItemGroup>
     <Reference Include="Microsoft.AspNetCore" />
     <Reference Include="Microsoft.AspNetCore.Components.WebAssembly.Server" />
-    <Compile Include="$(SharedSourceRoot)CommandLineUtils\**\*.cs" />
```

Comment:
I'd like to eventually delete CommandLineUtils.

Reply:
We have a couple of different command-line utilities but _something_ is needed to make our argument parsing relatively straightforward. I'm all for reducing the concept count but what choice are you suggesting we stick with❔